devicon.json

```
@@ -9111,5 +9111,37 @@
         },
         "color": "#2684FF",
         "aliases": []
+    },
+    {
+        "name": "pfsense",
+        "altnames": [
+            "pfsense"
+        ],
+        "tags": [
+           "network"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#FFFFFF",
```

Their original logo color is `#000000` even on their site it's `#ffffff`, so here's some suggestion.
```suggestion
        "color": "#000000",
```